bundles/org.openhab.binding.lghombot/src/main/java/org/openhab/binding/lghombot/internal/LGHomBotHandler.java

```diff
@@ -0,0 +1,722 @@
+                        if (!turbo.equals(currentTurbo)) {
+                            currentTurbo = turbo;
+                            channel = new ChannelUID(getThing().getUID(), CHANNEL_TURBO);
+                            updateState(channel, turbo);
+                        }
+                        break;
+                    case "JSON_REPEAT":
+                        OnOffType repeat = value.equalsIgnoreCase("true") ? OnOffType.ON : OnOffType.OFF;
+                        if (!repeat.equals(currentRepeat)) {
+                            currentRepeat = repeat;
+                            channel = new ChannelUID(getThing().getUID(), CHANNEL_REPEAT);
+                            updateState(channel, repeat);
+                        }
+                        break;
+                    case "JSON_MODE":
+                        if (!value.equals(currentMode)) {
+                            currentMode = value;
+                            channel = new ChannelUID(getThing().getUID(), CHANNEL_MODE);
+                            updateState(channel, StringType.valueOf(value));
+                        }
+                        break;
+                    case "JSON_NICKNAME":
+                        if (!value.equals(currentNickname)) {
+                            currentNickname = value;
+                            channel = new ChannelUID(getThing().getUID(), CHANNEL_NICKNAME);
+                            updateState(channel, StringType.valueOf(value));
+                        }
+                        break;
+                    case "CLREC_LAST_CLEAN":
+                        try {
+                            final String stringDate = value.substring(0, 19);
+                            LocalDateTime localDateTime = LocalDateTime.parse(stringDate, formatterLG);
+                            ZonedDateTime date = ZonedDateTime.of(localDateTime, ZoneId.systemDefault());
+                            DateTimeType lastClean = new DateTimeType(date);
+                            if (!lastClean.equals(currentLastClean)) {
+                                currentLastClean = lastClean;
+                                channel = new ChannelUID(getThing().getUID(), CHANNEL_LAST_CLEAN);
+                                updateState(channel, lastClean);
+                            }
+                        } catch (IndexOutOfBoundsException | DateTimeException e) {
+                            logger.debug("Couldn't parse Last Clean. String length: {}", value.length());
+                        }
+                        break;
+                    default:
+                        break;
+                }
+            }
+            if (!parsingOk) {
+                logger.debug("Couldn't parse response;\n {}", status);
+            }
+        }
+    }
+
+    private void fetchSchedule() {
+        String status = null;
+        String url = buildHttpAddress("/.../usr/data/htdocs/timer.txt");
```

Comment:
This looks like even a bigger hack... :-(

Reply:
This method should also re-use the command exec imho

Reply:
> This looks like even a bigger hack... :-(

I mean bigger then hacking/modding the HomBot now also iterating some directory structure starting from root, nothing about the code here 😄

Reply:
Yeah, this is how the web page served from the server does it... I'll try to use the sendCommand.